Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,19 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    rm_f "#{bin}/sndfile-resample" if build.without? "libsndfile"
+  end
+
+  def caveats
+    s = ""
```

Comment:
If there's no specific reason for the empty string I'm happy for it to be changed, but it's nice to be consistent so we don't end up with a weird mix and contributors aren't sure what to use, so let's do 'em all if a decision is made to rewrite them that way :smile:.